src/graphql/resolvers/userResolver.js

```diff
@@ -15,9 +16,37 @@ const resolver = {
 
     User: {
         async stockTransactions(parent, args) {
-            const stockTransactions = await StockTransaction.find({user: parent._id})
+            const stockTransactions = await StockTransaction.find({ user: parent._id })
             return stockTransactions
         }
+    },
+
+    Mutation: {
+        async register(_, { name, password, confirmPassword, email, bits_id }, context, info) {
```

Comment:
Remove confirmPassword from the argument list. That check is supposed to be done at the client side.